### ISSUE TITLE
Horizontal scrolling compability

### ISSUE THREAD

**Issue by LordotU** [mrphilipandrews/jquery.smoothwheel#2]
Hello!

Do you have any ideas how to make your plugin compatible with jquery.mousewheel and horizontal scrolling?

Thanks.

**Comment by Randore** [comment]
@LordotU I also wanted ask the same question, is this script compatible with horizontal scrolling?

**Comment by jeanmichelcote** [comment]
Indeed, would be quite interesting if somebody could craft up a blended version of jquery.smoothwheel.js and <a href="https://css-tricks.com/examples/HorzScrolling/jquery.mousewheel.js">jquery.mousewheel.js</a>
